polar-wasm-api/tests/polar.rs

```diff
@@ -33,10 +33,10 @@ fn load_file_errors() {
     let err = polar.wasm_load(sources).unwrap_err();
     let err: Error = err.dyn_into().unwrap();
     assert_eq!(err.name(), "ParseError::UnrecognizedToken");
-    assert_eq!(
-        err.message(),
-        "did not expect to find the token ';' at line 1, column 1"
-    );
+    assert!(err.message().starts_with(
+        "did not expect to find the token ';' at line 1, column 1",
+        0
```

Comment:
what's this 0 doing here?

Reply:
Oh, it's calling the JS method! Huh

Reply:
https://docs.rs/js-sys/0.3.44/js_sys/struct.JsString.html#method.starts_with